cmd/buildah/config.go

```diff
@@ -18,58 +18,58 @@ const (
 
 var (
 	configFlags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "annotation, a",
+			Usage: "add `annotation` e.g. annotation=value, for the target image",
+		},
 		cli.StringFlag{
-			Name:  "author",
-			Usage: "image author contact `information`",
+			Name:  "arch",
+			Usage: "set `architecture` of the target image",
 		},
 		cli.StringFlag{
-			Name:  "created-by",
-			Usage: "`description` of how the image was created",
-			Value: DefaultCreatedBy,
+			Name:  "author",
+			Usage: "set image author contact `information`",
 		},
 		cli.StringFlag{
-			Name:  "arch",
-			Usage: "`architecture` of the target image",
+			Name:  "cmd",
+			Usage: "Sets the default `command` to run for containers based on the image",
```

Comment:
ugh, hate to catch this now, but "sets the default"  lower case 's'

Reply:
Fixed